tests/gui/models/flash-state.spec.js

```diff
@@ -441,6 +470,52 @@ describe('Browser: flashState', function() {
 
     });
 
+    describe('.getFlashUuid()', function() {
+
+      const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/;
```

Comment:
Minor nit, but perhaps this should be `UUID_V4_REGEX`

EDIT: forget that, I've just seen they match the same regex ;-)  https://github.com/kelektiv/node-uuid